tests/e2e/s3/utils.py

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env python
+
+import subprocess
+import json
+import os
+import sys
+import boto3
+import time
+
+
+def update_agent_biling_clients(billing):
+    config = subprocess.check_output(
```

Comment:
My python's a little rusty — will `subprocess.check_output` raise an exception if the command exits with a non-zero exit code?